feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature1 branch
```

Comment:
It's really nice that you have decicded to write this code

Reply:
Thank for your comment